build/cekit/image.yaml

```diff
@@ -0,0 +1,50 @@
+- name: "builder-infinispan-operator"
+  description: "Builder for Infinispan Operator"
+  version: latest
+  from: "fedora:29"
+  labels:
+    - name: "maintainer"
+      value: "jboss-support-dg6@redhat.com"
+    - name: "com.redhat.component"
+      value: "jboss-datagrid-7-datagrid73-operator-container"
+    - name: "io.k8s.description"
+      value: "Operator for deploying RHDG"
+    - name: "io.k8s.display-name"
+      value: "Red Hat Data Grid Operator"
+    - name: "io.openshift.tags"
+      value: "datagrid,java,jboss,xpaas,operator"
+    - name: "com.redhat.delivery.appregistry"
+      value: "true"
+  modules:
+    repositories:
+      - path: modules
+        name: modules
+    install:
+      - name: builder-infinispan-operator
+        #osbs:
```

Comment:
You can remove commented sections from here. They should be added in product overrides.

Reply:
done. thanks @galderz !